includes/Checker/Preparations/Universal_Runtime_Preparation.php

```diff
@@ -54,7 +54,14 @@ public function prepare() {
 
 		$cleanup_functions = array();
 
-		$use_minimal_theme_preparation = new Use_Minimal_Theme_Preparation( 'wp-empty-theme', WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . '/test-content/themes' );
+		if ( ! defined( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH' ) ) {
+			$plugins_dir  = defined( 'WP_PLUGIN_DIR' ) ? WP_PLUGIN_DIR : WP_CONTENT_DIR . '/plugins';
+			$theme_folder = $plugins_dir . '/plugin-check/test-content/themes';
+		} else {
+			$theme_folder = WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . 'test-content/themes';
+		}
```

Comment:
This logic is inverted compared to https://github.com/10up/plugin-check/pull/101/files#diff-35a6c4633cd89c9ad66535db741d064ff3f730cf69ea87d347d8e250544fff1eR85